databricks-catalog-external-location/variables.tf

```diff
@@ -18,6 +18,22 @@ variable "external_location_name" {
   type        = string
 }
 
+variable "override_bucket_name_prefix" {
```

Comment:
I think this would be cleaner if you set this to default null, then adjusted the above logic to:
```
var.override_bucket_name_prefix != null ? <a> : <b>
```